### PR TITLE
[lldp_neighbor]: Add proper tests for system name, system description, port ID and port description

### PR DESCRIPTION
**Pull request by jleveque**
- Chassis ID test still only check for empty field; will require more work to implement proper test.

### CODE REVIEW

File: ansible/roles/test/tasks/lldp_neighbor.yml [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
+    community: "{{ snmp_rocommunity }}"
   connection: local
 
 - name: Print LLDP facts from neighbors
   debug: msg="{{ ansible_lldp_facts }}"
 
-# FIXME: use more strict assertions
-- name: verify the dut system name field is not empty
-  assert: {that: "'{{ ansible_lldp_facts[neighbor_interface]['neighbor_sys_name'] }}' != ''"}
+- name: Verify the published DuT system name field is correct
```
Comment (qiluo-msft) [inline ansible/roles/test/tasks/lldp_neighbor.yml]: DUT is more common than DuT.

File: ansible/roles/test/tasks/lldp_neighbor.yml [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+  lldp_facts:
+    host: "{{ hname }}"
+    version: "v2c"
+    community: "{{ snmp_rocommunity }}"
   connection: local
 
 - name: Print LLDP facts from neighbors
   debug: msg="{{ ansible_lldp_facts }}"
 
-# FIXME: use more strict assertions
```
Comment (qiluo-msft) [inline ansible/roles/test/tasks/lldp_neighbor.yml]: Is it easy/possible to FIXME now?

File: ansible/roles/test/tasks/lldp_neighbor.yml (hunk 1 above)
Comment (jleveque) [inline ansible/roles/test/tasks/lldp_neighbor.yml]: Resolved.

File: ansible/roles/test/tasks/lldp_neighbor.yml (hunk 2 above)
Comment (jleveque) [inline ansible/roles/test/tasks/lldp_neighbor.yml]: I figured out a fairly straightforward way to test the system description, and just made a new commit. Chassis ID, however, can take a few forms, and will require more advanced logic and parsing.

File: ansible/roles/test/tasks/lldp.yml [hunk 3]
```diff
@@ -29,7 +30,17 @@
   with_items: "{{ lldp.keys() }}"
   when: item != "eth0"
 
-- name: Iterate through each LLDP neighbor and verify the information received by neighbor are also correct
+- block:
+  - name: Obtain the system description of the DUT chassis
+    become: true
```
Comment (qiluo-msft) [inline ansible/roles/test/tasks/lldp.yml]: No need to become

Comment (jleveque) [inline ansible/roles/test/tasks/lldp.yml]: Thanks. Resolved.

**Pull request merged**